lib/ansible/modules/packaging/os/yum_repository.py

```diff
@@ -128,6 +128,8 @@
     description:
       - Tells yum whether or not it should perform a GPG signature check on
         packages.
+      - Doesn't add anything to config, by default, which implies fallback
```

Comment:
This description is wrong. It DOES add parameter into the repo file if the option IS specified.

By my opinion, it is not necessary to describe that there is an internal YUM default as that's absolutely clear if this option has no default value.

Reply:
Yes, it does add parameter, if the option is specified. But the description says "by default" and like you said it has no default value so it does NOT add that parameter, by default, like the description says. Seems correct to me.

Reply:
I am fine with not adding that note if you prefer, but it doesn't hurt either.

Reply:
@webknjaz Please remove this commit and we are good to merge it.